src/main/java/com/google/jenkins/plugins/computeengine/PreemptedCheckCallable.java

```diff
@@ -58,7 +58,18 @@
    */
   @Override
   public Boolean call() throws IOException {
-    HttpRequest request = createMetadataRequest();
+    HttpRequest initialRequest = createMetadataRequest(false);
```

Comment:
This came up in #148 as well. Having the preemption event occur before the agent is running and listening for preemption events is rare, but given that it can and does happen it makes sense to check that it's not already preempted before beginning the long poll to wait for a change to the metadata (which won't happen in that case).

Reply:
Makes sense.